### DISCUSSION TITLE
Multi label eval_metrics

### DISCUSSION THREAD

**Discussion by leoin841023** [autogluon/autogluon#1307] [Q&A] [answered]
I’m using version0.2.0 for doing multi label prediction,
In the document ,it said that eval_metrics should setting on multipredictor.fit()
But it can’t work….
Can anyone let me know the right code for setting eval_metrics on multi label prediction?
Thanks!

**Comment by Innixma** [comment]
@jwmueller Could you provide guidance on this?

**Comment by jwmueller** [comment]
@leoin841023  Based on your screenshot: https://user-images.githubusercontent.com/34364874/133010977-9840ef7f-ac02-4e0f-89cc-f48474a59108.jpg
You defined your list variable as `eval_metrix`, but then passed in `eval_metrics = eval_metric` (instead of `eval_metrix`) into `MultilabelPredictor()`

so I'm not sure your `eval_metric` is the same thing as `eval_metrix`?  It looks to me like you maybe accidentally passed in just a single metric instead of a list into `eval_metrics`.

Also note you can just omit the `eval_metrics` argument if you plan to set them all to `'root_mean_squared_error'` as this is anyway the default metric for regression.

**Comment by jwmueller** [comment, marked as answer]
@leoin841023 I did find an eval_metrics related bug that may also be causing your issue, I've fixed it in this PR:
https://github.com/awslabs/autogluon/pull/1314

You may copy the MultilabelPredictor class from there are retry your run now already, the new file is here:
https://github.com/awslabs/autogluon/blob/f923233d902db692aa10195c99163c11106a0da0/docs/tutorials/tabular_prediction/tabular-multilabel.md

**Reply by leoin841023** [reply]
Thanks for your quick reply! 
I think I uploaded the wrong screenshot, which made you misunderstood. Sorry!
It works now
